src/Nordea/Components/Range.elm

```diff
@@ -0,0 +1,153 @@
+module Nordea.Components.Range exposing
+    ( Config
+    , Range
+    , init
+    , view
+    , withShowInterval
+    )
+
+-- CONFIG
+
+import Css
+    exposing
+        ( Style
+        , backgroundColor
+        , borderRadius
+        , cursor
+        , displayFlex
+        , flexDirection
+        , height
+        , justifyContent
+        , margin2
+        , marginTop
+        , pct
+        , pointer
+        , property
+        , pseudoElement
+        , rem
+        , row
+        , spaceBetween
+        , transparent
+        , width
+        )
+import Html.Styled as Html exposing (Attribute, Html, div, input, span)
+import Html.Styled.Attributes exposing (css, name, type_)
+import Html.Styled.Events exposing (onInput)
+import Nordea.Html exposing (showIf)
+import Nordea.Resources.Colors as Colors
+import Nordea.Themes as Themes
+
+
+type alias Config msg =
+    { value : Float
+    , min : Float
+    , max : Float
+    , step : Maybe Float
+    , onInput : String -> msg
+    , showInterval : Bool
+    }
+
+
+type Range msg
+    = Range (Config msg)
+
+
+init : Float -> Float -> Float -> (String -> msg) -> Range msg
+init value min max onInput =
+    Range
+        { value = value
+        , min = min
+        , max = max
+        , step = Nothing
+        , onInput = onInput
+        , showInterval = False
+        }
+
+
+withShowInterval : Bool -> Range msg -> Range msg
+withShowInterval value (Range config) =
+    Range { config | showInterval = value }
+
+
+
+-- VIEW
+
+
+view : List (Attribute msg) -> Range msg -> Html msg
+view attributes (Range config) =
+    div attributes
+        [ input
+            [ name "rangeInput"
+            , type_ "range"
+            , Html.Styled.Attributes.value (config.value |> String.fromFloat)
+            , Html.Styled.Attributes.min (config.min |> String.fromFloat)
+            , Html.Styled.Attributes.max (config.max |> String.fromFloat)
+            , Html.Styled.Attributes.step (config.step |> Maybe.map String.fromFloat |> Maybe.withDefault "1")
+            , config.onInput |> onInput
+            , css [ sliderStyle (Range config) ]
+            ]
+            []
+        , showIf config.showInterval
+            (div [ css [ displayFlex, flexDirection row, justifyContent spaceBetween, margin2 (rem 1) (rem 0.25) ] ]
+                (List.range (config.min |> ceiling) (config.max |> ceiling) |> List.map (\number -> span [] [ Html.text (String.fromInt number) ]))
+            )
```

Comment:
Not 100% sure about the change to ol/li being semantically correct here. What do you think?
```suggestion
        , ol 
            [ css 
                [ displayFlex
                , flexDirection row
                , justifyContent spaceBetween
                , margin2 (rem 1) (rem 0.25) 
                ] 
            ]
            (List.range (ceiling config.min) (ceiling config.max) 
                |> List.map (\number -> li [] [ Html.text (String.fromInt number) ])
            )
            |> showIf config.showInterval
```

Reply:
I think its atleast more semantically correct than the div and spans right now either way 😅 Good suggestion!